src/components/color-extractor.ts

```diff
@@ -25,6 +25,10 @@ export class ColorExtractor {
 
   extractColorVariables(): void {
     let colorSystem = figma.currentPage.findChild(node => node.name === this.NODE_NAMES.colorSystem);
+
+    if (colorSystem === null) {
```

Comment:
There's a cooler way to check `null` hehe (variable in `if` clause will become a truth value)
```suggestion
    if (!colorSystem) {
```